docs/src/install.rst

```diff
@@ -74,53 +70,51 @@ of computers, in :doc:`distributed`, you should install with::
 
     easy_install gensim[distributed]
 
-The optional `distributed` feature installs `Pyro (PYthon Remote Objects) <http://pypi.python.org/pypi/Pyro>`_.
-If you don't know what distributed computing means, you can ignore it:
-`gensim` will work fine for you anyway.
+The optional ``distributed`` feature installs `Pyro (PYthon Remote Objects) <http://pypi.python.org/pypi/Pyro>`_.
+If you don't know what distributed computing means, you can ignore it: Gensim will work fine for you anyway.
+
 This optional extension can also be installed separately later with::
 
-    easy_install Pyro4
+    pip install Pyro4
 
 -----
 
 There are also alternative routes to install:
 
 1. If you have downloaded and unzipped the `tar.gz source <http://pypi.python.org/pypi/gensim>`_
-   for `gensim` (or you're installing `gensim` from `github <https://github.com/piskvorky/gensim/>`_),
+   for Gensim (or you're installing Gensim from `Github <https://github.com/piskvorky/gensim/>`_),
    you can run::
 
      python setup.py install
 
-   to install `gensim` into your ``site-packages`` folder.
-2. If you wish to make local changes to the `gensim` code (`gensim` is, after all, a
-   package which targets research prototyping and modifications), a preferred
-   way may be installing with::
+   to install Gensim into your ``site-packages`` folder.
+2. If you wish to make local changes to the Gensim code, a preferred way may be installing with::
 
      python setup.py develop
 
+   or::
+
+     pip install -e .
+
    This will only place a symlink into your ``site-packages`` directory. The actual
    files will stay wherever you unpacked them.
-3. If you don't have root priviledges (or just don't want to put the package into
-   your ``site-packages``), simply unpack the source package somewhere and that's it! No
-   compilation or installation needed. Just don't forget to set your PYTHONPATH
-   (or modify ``sys.path``), so that Python can find the unpacked package when importing.
 
 
-Testing `gensim`
-----------------
+Testing Gensim
+--------------
 
 To test the package, unzip the `tar.gz source <http://pypi.python.org/pypi/gensim>`_ and run::
 
     python setup.py test
 
-Gensim uses Travis CI for continuous integration: |Travis|_
+Gensim uses Travis CI for continuous integration, automatically running the full test suite on each pull request and commit: |Travis|_
```

Comment:
In addition
- Appveyour (windows) and CircleCI (documentation)
- https://github.com/MacPython/gensim-wheels (python wheels)
- https://github.com/conda-forge/gensim-feedstock (conda packages)
- https://github.com/RaRe-Technologies/gensim-data (datasets & pre-trained models)